docs/topics/secured-mission-nodejs-adapter.adoc

```diff
@@ -0,0 +1,50 @@
+= SSO Adapter Configuration
+
+The SSO adapter is the _client side_, or client to the SSO server, component that enforces security on the web resources. In this specific case, it is the Greeting service.
+
+
+.Enacting Security Example {NodeJS} Code
+[source,javascript,options="nowrap",subs="attributes+"]
+----
+const express = require(‘express’);
+const Keycloak = require(‘keycloak-connect’); <1>
```

Comment:
I'm unsure if it's convention or not in these docs, but I personally don't like the line numbers on the right here, unless asciidoc modifies this in some way. It makes it harder to copy/paste the code from a browser. You might consider making these line numbers actual comments. E.g.

```js
const kc = new Keycloak(); // <2>
```

Reply:
Asciidoctor converts that into a reference number that is explained under the block. For actual usage, see e. g. our [Contributor Guide](https://appdev.openshift.io/docs/contrib-guide.html#_repository_filesystem_layout).